src/scene/graph-node.js

```diff
@@ -1177,7 +1185,35 @@ pc.extend(pc, function () {
                 if (this._parent === null) {
                     this.worldTransform.copy(this.localTransform);
                 } else {
-                    this.worldTransform.mul2(this._parent.worldTransform, this.localTransform);
+                    if (this.scaleCompensation) {
+                        var parent = this._parent;
+
+                        // Find a parent of the first uncompensated node up in the hierarchy and use its scale * localScale
+                        var parentToUseScaleFrom = parent;
+                        while(parentToUseScaleFrom.scaleCompensation && parentToUseScaleFrom) {
```

Comment:
Hi @guycalledfrank
If parentToUseScaleFrom is null or undefined, then occur exception at line 1193 and the other.
Is parentToUseScaleFrom check really necessary ?